components/threading/global_threads/thread_item/thread_item.tsx

```diff
@@ -186,17 +189,22 @@ function ThreadItem({
                         )}
                     </div>
                 )}
-                <span>{postAuthor}</span>
-                {Boolean(channel) && (
-                    <Badge
-                        className={classNames({
-                            Badge__hidden: postAuthor === channel?.display_name,
-                        })}
-                        onClick={goToInChannelHandler}
-                    >
-                        {channel?.display_name}
-                    </Badge>
-                )}
+                <div className='ThreadItem__author'>{postAuthor}</div>
+                <div className='d-flex align-items-center'>
+                    {Boolean(channel) && postAuthor !== channel?.display_name && (
+                        <Badge onClick={goToInChannelHandler}>
+                            {channel?.display_name}
+                        </Badge>
+                    )}
+                    {isPostPriorityEnabled && (
+                        post?.props?.priority && (
+                            <PriorityBadge
+                                className={postAuthor === channel?.display_name ? 'ml-2' : ''}
+                                priority={post.props.priority}
+                            />
+                        )
+                    )}
```

Comment:
With the restructuring of `Badges` (porting to `Tag` component) I removed all of the manual spacing and replaced it with flex containers and the `gap` property. See this PR: #11100 

```suggestion
                    {isPostPriorityEnabled && post?.props?.priority && (
                        <PriorityBadge priority={post.props.priority}/>
                    )}
```